distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh

```diff
@@ -18,7 +18,6 @@ cd `dirname $0`
 PASSWORD="changeit"
 KEYFILE="../keystores/serverKeystore.jks"
 KEYTYPE="JKS"
-CLASSNAME="org.codice.ddf.security.certificate.generator.CertificateCommand"
```

Comment:
Change the shade plugin configuration to generate an executable JAR file. Class path/name no longer necessary,